github/teams.go

```diff
@@ -796,6 +796,18 @@ func (s *TeamsService) RemoveTeamProjectBySlug(ctx context.Context, org, slug st
 	return s.client.Do(ctx, req, nil)
 }
 
+// ListIDPGroupsOptions specifies the optional parameters to the ListIDPGroupsInOrganization method.
+type ListIDPGroupsOptions struct {
+	// The number of results to include per page.
+	PerPage int `url:"per_page,omitempty"`
+
+	// Page token.
+	Page string `url:"page,omitempty"`
+
+	// Filters the results to return only those that begin with the value specified by this parameter.
+	Query string `url:"q,omitempty"`
```

Comment:
```suggestion
	// Filters the results to return only those that begin with the value specified by this parameter.
	Query string `url:"q,omitempty"`
	
	ListOptions
```

Reply:
Hi! That's what I wanted to do at first, but unfortunately `ListOptions` doesn't fit here because its `Page` field is of type int and this endpoint uses a _string_ token. `ListCursorOptions` that was here before would do, but then it has all other cursor-related fields that this API does not accept. So does this situation warrant a new specific type? Or should we still reuse `ListCursorOptions`?

Reply:
Oh, right!  I apologize.  Yes, please use `ListCursorOptions` in this case.  
Thank you for catching my mistake!  I appreciate it.

Reply:
No problem. Done in e492cdd